Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -532,11 +535,15 @@ - (void)enterFullScreen
 
     _fullScreenState = WebKit::WaitingToEnterFullScreen;
     _blocksReturnToFullscreenFromPictureInPicture = manager->blocksReturnToFullscreenFromPictureInPicture();
+    _originalWindowSize = [webView window].frame.size;
 
     _window = adoptNS([[UIWindow alloc] initWithWindowScene:[[webView window] windowScene]]);
     [_window setBackgroundColor:[UIColor clearColor]];
     [_window setWindowLevel:UIWindowLevelNormal - 1];
     [_window setHidden:NO];
+#if HAVE(UIKIT_WEBKIT_INTERNALS)
+    [_window setFrame:CGRectMake(0, 0, 960, 540)];
```

Comment:
Possibly this should come from a constant or something? It's weird that it's just hardcoded, though I get that that was the idea.